src/main/java/com/shanebeestudios/mcdeop/Processor.java

```diff
@@ -75,6 +79,7 @@ public void init() {
             long finish = Duration.ofMillis(System.currentTimeMillis() - start).toMinutes();
             Logger.info("Process finished in %s minutes!", finish);
             if (app != null) {
+                app.toggleControls();
```

Comment:
This seems alright if compiling is successful, but where are you handling failure?

Reply:
You're right, I accidentally managed to ignore this if there is a runtime exception.